src/details/ArborX_DetailsMortonCode.hpp

```diff
@@ -15,7 +15,7 @@
 #include <ArborX_DetailsKokkosExtMinMaxOperations.hpp> // min. max
 #include <ArborX_GeometryTraits.hpp>
 
-#include <Kokkos_Core.hpp>
+#include <Kokkos_Core.hpp>  // abort
```

Comment:
Damn I used Kokkos style two spaces before the comment but it looks like we only use one in arborx

Reply:
While you are at it do you mind changing `// min. max` to `// min, max`?  It somehow bothers me